bootstrap/bootstrap.ps1

```diff
@@ -2,6 +2,7 @@ param
 (
     [string]$RepositoryRoot,
     [string]$ToolsLocalPath,
+    [string]$CliLocalPath,
```

Comment:
While we are here it might be worth adding an optional parameter to allow overriding the "version" link directory path as well.